<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1059</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="25,75">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="file_tree_tab">
       <attribute name="title">
        <string>Dateien</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QScrollArea" name="file_tree_scroll">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>233</width>
             <height>523</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QTreeView" name="file_tree"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="add_folder_button">
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Ordner Hinzufügen...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="duplicate_list_tab">
       <attribute name="title">
        <string>Duplikate</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QScrollArea" name="duplicate_list_scrollarea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>233</width>
             <height>552</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QFrame" name="frame">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="duplicate_list_sort_by_label">
                 <property name="text">
                  <string>Sortieren nach:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="duplicate_list_sort_by">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <item>
                  <property name="text">
                   <string># Duplikate</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string># Behandelt</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>p-Hash</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QListView" name="duplicate_list"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="duplicate_detail_box">
      <property name="title">
       <string>Duplikate</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QScrollArea" name="duplicate_detail_scroll">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="duplicate_details_scrollarea">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>754</width>
            <height>514</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6"/>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="select_all_frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QCheckBox" name="select_all_checkbox">
            <property name="text">
             <string>Alle Auswählen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="num_selected_label">
            <property name="text">
             <string>0 Bilder ausgewählt</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="delete_selected">
            <property name="text">
             <string>Löschen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ignore_selected">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Ignorieren</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1059</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionNewProject"/>
    <addaction name="actionOpenProject"/>
    <addaction name="actionSaveProject"/>
    <addaction name="actionSaveProjectAs"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHilfe">
    <property name="title">
     <string>Hilfe</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuHilfe"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>Über</string>
   </property>
  </action>
  <action name="actionNewProject">
   <property name="text">
    <string>Neu</string>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="text">
    <string>Öffnen</string>
   </property>
  </action>
  <action name="actionSaveProject">
   <property name="text">
    <string>Speichern</string>
   </property>
  </action>
  <action name="actionSaveProjectAs">
   <property name="text">
    <string>Speichern unter...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Beenden</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
